src/lib/chips/chip-list.ts

```diff
@@ -3,17 +3,31 @@ import {
   ChangeDetectionStrategy,
   Component,
   ContentChildren,
+  Directive,
   Input,
   QueryList,
   ViewEncapsulation,
   OnDestroy,
+  ElementRef,
+  Renderer2,
 } from '@angular/core';
 
 import {MdChip} from './chip';
 import {FocusKeyManager} from '../core/a11y/focus-key-manager';
 import {coerceBooleanProperty} from '../core/coercion/boolean-property';
-import {SPACE, LEFT_ARROW, RIGHT_ARROW, TAB} from '../core/keyboard/keycodes';
 import {Subscription} from 'rxjs/Subscription';
+import {
+  LEFT_ARROW, RIGHT_ARROW, SPACE, BACKSPACE, DELETE, UP_ARROW, DOWN_ARROW
+} from '../core/keyboard/keycodes';
+import {Dir} from '../core/rtl/dir';
+
+@Directive({
+  selector: '[mdChipListContainer], [matChipListContainer]',
```

Comment:
One of the thing we've been deliberate about with chips is that the actual chip-list and the individual chips don't know that they're being driven by an `<input>`; that connection is managed by `mdChipInput`. I'd like to keep that separation of concerns. That's why I think it's a useful idea to _also_ have something that combines the behavior of both input-container and chip-list as a convenience to reduce boilerplate.